src/views/collection-view/index.tsx

```diff
@@ -1,29 +1,108 @@
 import CollectionMeta from './components/collection-meta'
 
 function CollectionView({
   collection,
-  product,
   layoutProps,
 }: CollectionPageProps): React.ReactElement {
-  const { name, slug, description, shortName, tutorials } = collection
+  const currentProduct = useCurrentProduct()
+  const currentPath = useCurrentPath({ excludeHash: true, excludeSearch: true })
+  const { name, slug, description, tutorials } = collection
+
+  // TODO: refactor, very long
+  const sidebarNavDataLevels = [
+    {
+      levelButtonProps: {
+        levelDownButtonText: `${currentProduct.name} Home`,
+      },
+      menuItems: generateTopLevelSidebarNavData(),
+      showFilterInput: false,
+      title: 'Main Menu',
+    },
+    {
+      levelButtonProps: {
+        levelUpButtonText: 'Main Menu',
+        levelDownButtonText: 'Previous',
+      },
+      menuItems: generateProductLandingSidebarNavData(currentProduct),
+      showFilterInput: false,
+      title: currentProduct.name,
+    },
+    {
+      levelButtonProps: {
+        levelUpButtonText: `${currentProduct.name} Home`,
+        levelDownButtonText: 'Previous',
+      },
+      backToLinkProps: {
+        text: `${currentProduct.name} Home`,
+        href: `/${currentProduct.slug}`,
+      },
+      title: 'Tutorials',
+      children: [
+        <>
+          <SectionList
+            items={[
+              {
+                text: 'Overview',
+                href: `/${currentProduct.slug}/tutorials`,
+                isActive: currentPath === `/${currentProduct.slug}/tutorials`,
+              },
+            ]}
+          />
+          {layoutProps.collectionViewSidebarSections.map(
+            (section: CollectionCategorySidebarSection) => {
+              return (
+                <>
+                  <HorizontalRule />
+                  <SectionTitle text={section.title} />
+                  <SectionList items={section.items} />
+                </>
+              )
+            }
+          )}
+          <HorizontalRule />
+          <SectionTitle text="Resources" />
+          <SectionList
+            items={[
+              {
+                text: 'All Tutorials',
+                href: 'https://learn.hashicorp.com',
+              },
+              {
+                text: 'Community Forum',
+                href: 'https://discuss.hashicorp.com',
+              },
+              { text: 'Support', href: 'https://support.hashicorp.com' },
+              { text: 'GitHub', href: 'http://github.com/hashicorp' },
+            ]}
+          />
+        </>,
+      ],
+    },
+  ]
 
   return (
     <SidebarSidecarLayout
       breadcrumbLinks={layoutProps.breadcrumbLinks}
       headings={layoutProps.headings}
-      sidebarSlot={
-        <ProductCollectionsSidebar
-          product={{ name: product.name, slug: product.slug }}
-          sections={layoutProps.sidebarSections}
-        />
-      }
+      AlternateSidebar={TutorialsSidebar}
```

Comment:
Could you walk me through the function of `AlternateSidebar` now?

Reply:
So it's a more structured version of `sidebarSlot`, and doesn't let just anything to be rendered in the `Sidebar` portion of the layout. It's currently pretty manual (depends on the developer to know exactly what to do) and I'm not sure yet how/if the TypeScript can be updated to enforce the rules the `AlternateSidebar` expects the component passed to it to follow. But the gist is: the name of the component passed to `AlternateSidebar` needs to render a `Sidebar` inside of it.

Reply:
Got it, and the idea that were retaining `TutorialSidebar` (as the alternate passed) even though it is effectively the same as `Sidebar` is that later on we may adjust how this alternate sidebar behaves?